test/integration/testConnection.js

```diff
@@ -1159,7 +1159,7 @@ describe('Connection Test - Heartbeat', () =>
   it('call heartbeat url as promise', async () =>
   {
     const rows = await connection.heartbeatAsync();
-    assert.ok(rows.success);
+    assert.deepEqual(rows,  [ { '1': 1 } ]);
```

Comment:
In JDBC we have certain tests for latest driver and few tests for older driver. Do you think we can do the same here, if we are running some tests with older version?

Reply:
do you mean running such compatibility tests during each build or have such tests running only on upcoming release? we can think about such tests to ensure that everything is fine between releases

Reply:
For each build. i.e. for each PR.